images/users/snapdeal.svg

```diff
@@ -0,0 +1 @@
 0-18.83 6.29-18.83 10.446 0 1.015.302 2.03 1.007 2.74l.2.202c.706.71 1.51 1.014 2.52 1.014 1.71 0 2.92-1.318 4.53-2.94 2.014-2.03 5.236-4.16 10.573-4.16 7.35 0 11.783 4.16 11.783 11.056v2.536h-13.092c-12.688 0-20.14 5.78-20.14 15.822 0 9.433 7.15 16.23 17.22 16.23 7.15 0 12.69-2.84 16.113-7.405v2.23c0 2.03 1.71 3.754 3.826 3.754h.2c2.015 0 3.728-1.724 3.728-3.853V44.324c0-11.46-7.35-18.358-19.637-18.358zM349.904 56.8c-1.007 9.33-5.64 14.808-14.098 14.808-9.566 0-14.3-7.1-14.3-19.17 0-12.07 4.734-19.17 14.3-19.17 8.56 0 13.192 5.68 14.098 15.214V56.8zm7.855 7.404V4.057c0-2.13-1.713-3.854-3.828-3.854h-.202c-2.114 0-3.826 1.725-3.826 3.854v29.11c-3.525-4.666-8.76-7.202-15.71-7.202-13.393 0-20.743 9.738-20.743 26.473s7.35 26.473 20.745 26.473c7.45 0 12.99-2.94 16.414-8.316 1.41 4.26 4.732 7.607 7.954 7.607 2.115 0 3.928-1.52 3.928-3.752 0-4.057-4.733-3.45-4.733-10.244zM284.75 71.608c-9.566 0-14.3-7.1-14.3-18.764v-.812c0-11.663 4.734-18.763 14.3-18.763 9.365 0 14.3 7.1 14.3 19.168 0 12.07-4.935 19.17-14.3 19.17m1.61-45.643c-6.947 0-12.284 2.638-15.81 7.405v-2.232c0-2.028-1.71-3.753-3.725-3.753h-.2c-2.115 0-3.828 1.725-3.828 3.753V93.82c0 2.13 1.713 3.855 3.827 3.855h.2c2.116 0 3.828-1.725 3.828-3.855V71.608c3.524 4.767 8.76 7.303 15.71 7.303 13.392 0 20.744-9.736 20.744-26.472 0-16.735-7.352-26.473-20.745-26.473M388.372 33.268c8.963 0 13.494 5.376 13.998 14.81h-27.795c1.008-9.333 5.54-14.81 13.797-14.81zM408.11 67.55c-1.61-1.52-3.828-1.724-5.64-.304-3.928 3.043-7.452 4.362-12.487 4.362-9.667 0-14.904-6.187-15.508-16.533h30.915c2.92 0 5.036-2.13 5.036-5.07 0-15.013-7.15-24.04-22.054-24.04-14.5 0-22.053 9.84-22.053 26.473 0 16.634 8.155 26.473 23.663 26.473 7.352 0 12.487-1.824 17.926-6.288 1.51-1.217 1.61-3.65.3-4.97l-.1-.1zM172.367 73.636V48.584c0-9.535 4.733-15.316 13.595-15.316 8.56 0 12.286 4.868 12.286 14.606v25.762c0 2.13 1.71 3.854 3.826 3.854h.202c2.115 0 3.826-1.723 3.826-3.854V47.67c0-14.706-5.64-21.705-18.327-21.705-6.95 0-12.185 2.638-15.51 7.405v-2.23c0-2.03-1.71-3.755-3.724-3.755h-.2c-2.115 0-3.828 1.725-3.828 3.754v42.496c0 2.13 1.713 3.854 3.828 3.854h.2c2.115 0 3.827-1.723 3.827-3.854M474.17 78.2c2.115 0 3.927-1.52 3.927-3.752 0-4.057-4.733-3.45-4.733-10.244V4.057c0-2.13-1.712-3.854-3.827-3.854h-.2c-2.116 0-3.828 1.725-3.828 3.854v62.277c0 5.984 4.33 11.867 8.66 11.867" fill="#333"/><g transform="translate(0 4.91)"><mask id="b" fill="#fff"><use xlink:href="#a"/></mask><path d="M21.718 90.23c1.108 1.927 2.316 2.637 4.53 2.637h41.29c2.215 0 3.423-.61 4.833-2.435l31.22-40.57c.703-.914-.102-2.435-1.21-2.435H52.33c-2.214 0-3.423-.71-4.53-2.636L22.624.974c-.603-1.014-2.215-.81-2.618.203L.57 48.543c-.905 2.13-.704 3.55.404 5.477l20.744 36.21z" fill="#E40046" mask="url(#b)"/></g><path d="M73.68 42.803c1.108 1.927 2.316 2.637 4.53 2.637h24.17c1.208 0 2.115-1.522 1.51-2.637L80.83 2.638C79.723.71 78.514 0 76.3 0H30.175c-1.207 0-2.113 1.52-1.51 2.638L40.65 23.532c1.108 1.926 2.316 2.636 4.532 2.636h19.032l9.466 16.635z" fill="#E40046"/></g></svg>
```

Comment:
As mentioned at the bottom of the user page (http://babeljs.io/users), could you please submit a 500x200 svg.